### ISSUE TITLE
accessibility: keyboard shortcuts

### ISSUE THREAD

**Issue by thet** [plone/Products.CMFPlone#348]
i've been wondering since long, how the keyboard shortcuts on Plone's accessibility page ( /accessibility-info ) are supposed to work. how is this implemented? is it implemented at all?

Pressing 1,2,3...0 does not work, nor `<alt><1..0>` or `<ctrl><1...0>`. futhermore, there are browser and desktop/window manager shortcuts, which come in conflict with the ones proposed at the accessibility page.
trac seems to have an option to configure if the shortcuts should be available: https://dev.plone.org/wiki/TracAccessibility

**Comment by thet** [comment]
@polyester do you know how to use the shortcuts on the accessibility page?

**Comment by polyester** [comment]
current state-of-the-art is to *not* use these anymore, as they interfere with the shortcuts that people have set up in their own screenreaders. It is being actively discouraged, actually.

 The discussion within the a11y community is not 100% stable yet, but is definitely trending towards not having site-specific shortcuts anymore, but switch completely to correct, tested, WAI-aria roles so people can have their own shortcuts.

I fully believe this is what we should be striving to as well. I will try do to a PLIP on this in the next few weeks.

**Comment by polyester** [comment]
To make this even more stern: most modern a11y checkers will actually deduct points for having site-specific keyboard shortcuts. Not that those checkers are always right, but in this case they do have a point. We should get rid of the accessibility page, basically, and instead switch to treating a11y users as first-class citizens, having the roles in place, working and tested.

**Comment by thet** [comment]
good ideas. we should get at least the accessibility page right for plone 5.
looking forward for the WAI-aria PLIP.

**Comment by vangheem** [comment]
duplicate of https://github.com/plone/Products.CMFPlone/issues/627
